src/current/releases/index.md

```diff
@@ -113,21 +113,20 @@ As of 2024, CockroachDB is released under a staged delivery process. New release
 
 | Version | Release Type | GA date |
 | :---: | :---: | :---: |
+| [v25.1](#v25-1) | Innovation | 2025-02-18 |
 | [v24.3](#v24-3) | Regular | 2024-11-18 |
 | [v24.1](#v24-1) | Regular | 2024-05-20 |
-| [v23.2](#v23-2) | Regular | 2024-02-05 |
-| [v23.1](#v23-1) | Regular | 2023-05-15 |
```

Comment:
Thought it made sense to remove these since they are now EOS, wdyt @katmayb ?

Reply:
Probably best to have @mikeCRL confirm, but I thought these would stay because they are still with assistance support: https://deploy-preview-19549--cockroachdb-docs.netlify.app/docs/releases/release-support-policy#supported-versions?

Reply:
Added